source/data-modeling/associations.txt

```diff
@@ -515,3 +515,410 @@ method to remove all embedded ``Album`` documents from the ``Band`` class:
    :language: ruby
    :start-after: # start-embedded-destroy-all
    :end-before: # end-embedded-destroy-all
+
+Association Behaviors
```

Comment:
q: is 'behaviors' a technical term here, or just a general way to describe how associations work? if it's the latter, something like 'Customize Association Behavior' might be clearer.